.github/workflows/snyk.yml

```diff
@@ -17,5 +17,6 @@ jobs:
       DEBUG: true
       ORG: guardian-mobile
       SKIP_NODE: false
+      SKIP_SBT: true
```

Comment:
Last Snyk job [failed in main ](https://github.com/guardian/react-crossword/actions/runs/4345653064/jobs/7590665642)with error: 

```
Error: No file in /home/runner/work/react-crossword/react-crossword matched to [**/*.sbt,**/project/build.properties,**/project/**.{scala,sbt}], make sure you have checked out the target repository
```

Reply:
Triggered the job manually in this branch and it [succeeded](https://github.com/guardian/react-crossword/actions/runs/4345844339).

Reply:
Thanks - great spot!